indico_toolkit/snapshots/snapshot.py

```diff
@@ -214,6 +216,28 @@ def split_and_write_to_csv(
             rows_taken += split_length
             print(f"Wrote split {i} of {num_splits}: {split_file_loc}")
 
+    def update_snapshot_shape(self, task_type: str = "annotation"):
+        """
+        Mutates old snapshot df shape into updated df with spans.
+        Args:
+            task_type (str): Task type to specifiy df as (annotation = extraction, classification = classification)
+
+        NOTE: page_num in the span is currently set to None.
+        """
+        for i, targets in enumerate(self.df[self.label_col]):
+            updated_targets = []
+            for span in targets:
+                updated_span = {
+                    "label": span["label"],
+                    "spans": [
+                        {"start": span["start"], "page_num": None, "end": span["end"]}
+                    ],
+                }
+                updated_targets.append(updated_span)
+
+            updated_col = {"targets": updated_targets, "task_type": task_type}
+            self.df.at[i, self.label_col] = updated_col
```

Comment:
you may want to consider just appending "update_col" (which is actually just updated row value) to an empty list and then overriding the whole column when you're done vs. the index method-- if an exception where raised half way through, you'd end up with a 50% modified data frame. Likewise, we aren't concerned with memory constraints of temporarily storing the column data

Reply:
Adjust logic to override entire column when done with iteration. 

Resolved in commit [9090f41](https://github.com/IndicoDataSolutions/Indico-Solutions-Toolkit/pull/138/commits/9090f41f200a2c2f4fa60c0bb1107028052c7e0b)